src/buildApplication/buildApplication.ts

```diff
@@ -32,10 +32,17 @@ export async function buildApplication({
 	if (!skipBuild) {
 		try {
 			await buildVercelOutput();
-		} catch (err) {
+		} catch {
 			cliError(
-				(err as Error)?.message ??
-					'Error: The Vercel build failed. For more details see the Vercel logs above.'
+				`
+					The Vercel build command failed. For more details see the Vercel logs above.
```

Comment:
Could we inline the command so that people can see what was run? If I was in this position, my next step might be to `npx vercel build` and see the output directly.

Reply:
mh.... ok good point 🙂👍 , I'll add that in, although I hope that this doesn't get too verbose 😓